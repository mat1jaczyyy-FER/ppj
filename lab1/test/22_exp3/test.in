a = (3+5) / 12   * 17
 b = (a+a)+a*a - a - a
  c = a*b + b*a + b*b*b/a
