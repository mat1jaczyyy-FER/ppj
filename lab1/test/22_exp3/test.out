IDN 1 a
OP_PRIDRUZI 1 =
L_ZAGRADA 1 (
BROJ 1 3
OP_PLUS 1 +
BROJ 1 5
D_ZAGRADA 1 )
OP_DIJELI 1 /
BROJ 1 12
OP_PUTA 1 *
BROJ 1 17
IDN 2 b
OP_PRIDRUZI 2 =
L_ZAGRADA 2 (
IDN 2 a
OP_PLUS 2 +
IDN 2 a
D_ZAGRADA 2 )
OP_PLUS 2 +
IDN 2 a
OP_PUTA 2 *
IDN 2 a
OP_MINUS 2 -
IDN 2 a
OP_MINUS 2 -
IDN 2 a
IDN 3 c
OP_PRIDRUZI 3 =
IDN 3 a
OP_PUTA 3 *
IDN 3 b
OP_PLUS 3 +
IDN 3 b
OP_PUTA 3 *
IDN 3 a
OP_PLUS 3 +
IDN 3 b
OP_PUTA 3 *
IDN 3 b
OP_PUTA 3 *
IDN 3 b
OP_DIJELI 3 /
IDN 3 a
